### ISSUE TITLE
nib run command fails with node container

### ISSUE THREAD

**Issue by ben-dalton** [technekes/nib#140]
Command:
`nib run frontend npm install --save react-split-pane`
Error:
`ERROR: Cannot create container for service frontend: invalid volume spec "rw": invalid volume specification: 'rw': invalid mount config for type "volume": invalid mount path: 'rw' mount path must be absolute`

The same command works replacing `nib` with `docker-compose`.

Possibly an issue with nib expecting a `:` in mounted volumes. 
```  frontend:
    build: ./frontend
    volumes:
      - ./frontend:/usr/src/app
      - /usr/src/app/node_modules
```

@johnallen3d

**Comment by johnallen3d** [comment]
Thanks for the bug report @ben-dalton! I did a little research and this would appear to be a [bug upstream with `docker-compse`](https://github.com/docker/compose/issues/3664). It looks like [issue has been resolved](https://github.com/docker/compose/pull/4562) and was targeted at the `v1.12.0` release (which dropped [18 days ago](https://github.com/docker/compose/releases/tag/1.12.0)). It seems that Docker for Mac is currently shipping with `v1.11.2`. For the sake of testing I went ahead and installed `v1.12.0` and the problem you describe is resolved. I think the best course of action is to wait and drop down to `docker-compose` for this scenario in the meantime.
